playground/src/screens/sharedElementTransition/CocktailDetailsScreen.js

```diff
@@ -100,6 +100,9 @@ const styles = StyleSheet.create({
     height: SIZE,
     width: SIZE,
     zIndex: 1,
+    // transform: [
+    //   { rotate: '45deg' }
+    // ],
```

Comment:
```suggestion
```

Reply:
I actually left that for you, since it seems like rotate animation isn't implemented on iOS. Should we remove the comment?